Representable/representable
# Increasing accessiblity on the survey page

To test:
* run python manage.py runserver
* go through the survey flow
* check whether or not the buttons are the right size and aligned


screenshots***
